### PR TITLE
Modvend 86 - Merge EDI into Vendor record

### PR DESCRIPTION
**Pull request by komick**
## Purpose
Since each vendor has only ever a single EDI listed, the EDI models can be merged into the vendor record in order to simplify the model structure.

https://issues.folio.org/browse/MODVEND-86

## Approach
The APIs, unit tests and test data for EDIs, EDI_FTPs, and EDI_Jobs were all removed.   However, the sample vendor data did not need updating as the JSON structure for EDIs were kept consistent with the previous versions.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] Were any API paths or methods changed, added or removed?
  - [x] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [x] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [x] If not, please create them
  - [x] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [x] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [x] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [x] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/ContactsAPI.java [hunk 1]
```diff
@@ -41,31 +41,31 @@ private boolean isInvalidUUID (String errorMessage) {
...
 
-  public EdisAPI(Vertx vertx, String tenantId) {
+  public ContactsAPI(Vertx vertx, String tenantId) {
     PostgresClient.getInstance(vertx, tenantId).setIdField(idFieldName);
   }
 
 
   @Override
-  public void getVendorStorageEdis(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+  public void getVendorStorageContacts(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```
Comment (craigmcnally) [inline src/main/java/org/folio/rest/impl/ContactsAPI.java:38]: I thought we started using PgUtil in mod-vendors... or was that only in mod-orders-storage?  Regardless it should simplify the code a bit and should be considered...

**Review by craigmcnally — CHANGES_REQUESTED** [review]
Code coverage is low...

File: src/main/java/org/folio/rest/impl/ContactsAPI.java (hunk 1 above)
Comment (komick) [reply src/main/java/org/folio/rest/impl/ContactsAPI.java:38]: Those changes were made as part of the mod-vendors-85 PR, which will be merged in before this PR.  I had forked the off of the mod-vendor-85 branch before you suggested I make those changes.  So once both branches are merged in, the PgUtils will be used for unit tests.

Comment (craigmcnally) [reply src/main/java/org/folio/rest/impl/ContactsAPI.java:38]: that makes this a little confusing and harder to review... so what does that mean for code coverage?

Comment (craigmcnally) [reply src/main/java/org/folio/rest/impl/ContactsAPI.java:38]: Maybe you could rebase off your branch to pull those changes in here...

Comment (komick) [reply src/main/java/org/folio/rest/impl/ContactsAPI.java:38]: Code rebased and everything looks good

**Review by craigmcnally — APPROVED** [review]

File: src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -63,28 +63,7 @@
...
-    {
-      "tableName": "edi_ftp",
-      "generateId": true,
-      "fromModuleVersion": 1.0,
-      "withMetadata": true,
-      "populateJsonWithId": true
-    },
-    {
-      "tableName": "edi_job",
+      "tableName": "contact",
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/schema.json:66]: This file needs a little cleanup, there are few extra tables being created like vendor_category, contact_category that are not being used.

**Review by KVupp — COMMENTED** [review]
The get collection implementation was also extracted out in mod-orders-storage. I am not sure if it was decided to be handled in a different story. It would be nice to have that code in a single class.

Other than that I just have one minor in line comment.

**Pull request merged**